projects/storefrontlib/src/cms-components/product/product-list/container/product-list-component.service.ts

```diff
@@ -31,7 +31,7 @@ import { ProductListRouteParams, SearchCriteria } from './product-list.model';
 @Injectable({ providedIn: 'root' })
 export class ProductListComponentService {
   // TODO: make it configurable
-  protected defaultPageSize = 10;
+  protected defaultPageSize = 12;
```

Comment:
fixes the issue at hand, but I rather have this configurable as the comment says. I believe this was left out for a long time.

What we can do here is make it configurable by putting it in the `view-config.ts`, which the customer can configure to the size they want

Therefore, need to update

- `view-config.ts`
- `default-scroll-config.ts` --> change the name to default-view-config.ts` as we probably want a default size of 12.